@tc/eng 2195 docs reorg (#15258)
expo/expo: docs/pages/development/build.md

</Tab>
<Tab>

To allow iOS devices to run a build built for internal distribution, you'll have to register each iOS device you'd like to install your development build on.

You can register an iOS device and install a provisioning profile with the following command:
<TerminalBlock cmd={["eas device:create"]} />

Once you've registered all iOS devices you'll want to run your development build on, you can run the following to create a build ready for internal distribution:
<TerminalBlock cmd={["eas build --profile development --platform ios"]} />

To share the build with your team, direct them to the build page on https://expo.dev. There, they'll be able to download the app directly on their device.

Note: If you register any new iOS devices, you'll need create a new development build. Learn more about [internal distribution](https://docs.expo.dev/build/internal-distribution/).

</Tab>
</Tabs>

## Locally with Xcode and Android Studio

If you are comfortable setting up Xcode, Android Studio, and related dependencies, you can build and distribute your app the same as any other iOS or Android app.

> The `expo run` commands will run a new build, install it on to your emulated device, and launch you into your app.

Reviewer: ```suggestion
The `expo run` commands will make a new build, install it on your device or emulator, and start it running.
```

<Tabs tabs={["For Android", "For iOS (MacOS Only)"]}>

<Tab >

To build and run on an emulator
<TerminalBlock cmd={["expo run:android"]} />

To build and run on a connected device
<TerminalBlock cmd={["expo run:android -d"]} />

</Tab>
<Tab >

To build and run on a simulator
<TerminalBlock cmd={["expo run:ios"]} />

To build and run on a connected device
<TerminalBlock cmd={["expo run:ios -d"]} />
</Tab>

> 
> 

Reviewer 2: ```suggestion
```
🧹
